uaaextras/webapp.py

```diff
@@ -164,10 +164,16 @@ def send_email(app, email, subject, body):
     s = smtplib.SMTP(app.config['SMTP_HOST'], app.config['SMTP_PORT'])
     s.set_debuglevel(1)
 
-    s.starttls()
+    # if we have a cert, then trust it
+    if 'SMTP_CERT' in app.config and app.config['SMTP_CERT']:
+        sslcontext = ssl.create_default_context()
+        sslcontext.load_verify_locations(cadata=app.config['SMTP_CERT'])
+        s.starttls(context=sslcontext)
+    else:
+        s.starttls()
 
     # if smtp credentials were provided, login
-    if app.config['SMTP_USER'] is not None and app.config['SMTP_PASS'] is not None:
+    if 'SMTP_USER' in app.config and 'SMTP_PASS' in app.config and app.config['SMTP_USER'] and app.config['SMTP_PASS']:
```

Comment:
It looks like these keys are *always* set: https://github.com/18F/cg-uaa-extras/blob/712e6224dc19b810a649254ceb667f7831a0dca4/uaaextras/webapp.py#L25-L40. Do we need to check whether the keys are in the dict?

Reply:
Oh, the reason why I did this was because the logic that was being used for SMTP_USER/PASS didn't work for SMTP_CERT, so I got it working and just copied it over to be sure that it worked well too.

Reply:
Oh, I get it.  I didn't notice that None was what it was set to by default.  Interesting.  I will revert.  Thanks for pointing that out!